fastlane/Fastfile

```diff
@@ -175,6 +175,7 @@ end
   lane :new_beta_release do | options |
     ios_betabuild_prechecks(options)
     ios_update_metadata(options)
+    sanitize_appstore_keywords(options)
```

Comment:
I have to say, this "passing options to every lane" approach is making me feel very uneasy. I like things being deliberate and obvious, and this is neither. I don't think we even use `options` in the `sanitize_appstore_keywords`.

I am guessing this is because I am not a Ruby/fastlane developer and this is just how it is?

I've never been a fan of languages that don't have a strong type system and feel very discouraged even if I have to make a very small change like this. 😿

Reply:
It's not just you. I like explicit params too.

I think it's used that way Kb multiple places only because of the sheep effect 😅 it was done like this once or twice at first in our Fastfile, then all subsequent changes started to take example on it and now is everywhere even where it's not justified and explicit parameters would be better and more readable.

For this specific case if we don't need any of the options in the sanitize method then you should just omit it completely.

Reply:
Addressed in e51065d84476f35b1840019f1a2f725474085804.